src/main/kotlin/com/intellij/ml/llm/template/intentions/ApplyExtractFunctionTransformationIntention.kt

```diff
@@ -86,11 +87,13 @@ abstract class ApplyExtractFunctionTransformationIntention(
             project, LLMBundle.message("intentions.request.extract.function.background.process.title")
         ) {
             override fun run(indicator: ProgressIndicator) {
+                val now = System.currentTimeMillis()
```

Comment:
`System.currentTimeMillis()` may be inaccurate during clock synchronization, so I suggest using `System.nanoTime()`.

Reply:
Done